docs/rules/no-extra-label.md

```diff
@@ -0,0 +1,87 @@
+# Disallow Unnecessary Labels (no-extra-label)
+
+We don't need to use labels in order to control flow of the direct containing loop or switch statement.
```

Comment:
"directly labeled"? Not sure that's much clearer.

Maybe it should say, "If a loop contains no nested loops, labeling the loop is unnecessary" or similar?